packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js

```diff
@@ -28,7 +28,6 @@ const mapStateToProps = (state, ownProps) => ({
   currencies: [
     { text: 'EUR', value: 'EUR' },
     { text: 'DKK', value: 'DKK' },
-    { text: 'USD', value: 'USD' },
```

Comment:
This will also remove USD as an option for buy. This should only be removed for Sell.